### DISCUSSION TITLE
Plotting geographic data

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#1754] [Q&A] [answered]
I'd like to plot some geographic data to wow my audience, similarly to Saul's lightning talk. How do I do that really? I couldn't figure it out from the video.

**Comment by saulpw** [comment, marked as answer]
Make the longitude column a key column (x axis) and press `.` on the latitude column to get a Mercator projection. Both need to be floats, of course.

VisiData can also read Shapefiles (.shp) and you can press `.` to plot those maps too.

**Reply by cool-RR** [reply]
Thank you! Would that also work on a JSON file rather than GeoJSON?

**Reply by cool-RR** [reply]
Hmm, I tried it now with [this dataset](https://www.dropbox.com/s/pk9ca55ojf974ry/schools-list.geojson?dl=0) but the plot came out with just one dot. Any idea why?

**Reply by cool-RR** [reply]
This is one of the last few pieces I need for my talk.

**Reply by cool-RR** [reply]
Looks like it's a bug with GeoJSON and it works for normal files, so never mind.

**Reply by saulpw** [reply]
For GeoJSON files and shpfiles `.` only plots the current row; use `g.` to plot all rows.  (Yes this is wonky and we should change it).
